### PR TITLE
moving ephys module positions to ephys session

### PR DESCRIPTION
**Pull request by saskiad**
WIP to move the dome module to session (and deal with other items listed in issue #249

### CODE REVIEW

**Comment by saskiad** [comment]
Take a look at this and lmk what you think - I haven't dealt with tests and validators yet, just trying to get the logic right first

**Comment by jsiegle** [comment]
In `ephys_session.py`, `ManipulatorModule` needs to inherit from `DomeModule` (since every manipulator module also has a position on the dome). And there needs to be a separate `EphysModule` class (similar to `LaserModule`) that holds a `List` of `EphysProbes`, since there can be multiple probes linked to single manipulator.

**Comment by saskiad** [comment]
@jsiegle just clarifying that there can be multiple ephys probes in a manipulator. So multiple ephys probes can have the same DomeModule angles, etc, as well as the same ManipulatorModule coordinates? So this can be solved by making the EphysProbe class contain a list of probes, yes?

**Comment by jsiegle** [comment]
I think we still want to differentiate between `EphysProbe` (each of which generates a unique data stream) and `EphysModule` (which represents probes attached to the same manipulator). But yes, the probes on each module will share manipulator coordinates, target CCF coordinates, etc.

**Comment by saskiad** [comment]
cool - I'll work on the test and examples and let you know when it's ready for review

**Comment by saskiad** [comment]
@jsiegle ready for review now.

File: src/aind_data_schema/ephys/ephys_rig.py [hunk 1]
```diff
@@ -107,37 +107,24 @@ class OpenEphysAcquisitionBoard(DAQDevice):
...
     camera: Camera = Field(..., title="Camera for this module")
+    lens: Lens = Field(..., title="Lens for this module")
 
 
-class LaserModule(DomeModule):
-    """Module for optogenetic stimulation"""
+class LaserAssembly(AindModel):
+    """Assembly for optogenetic stimulation"""
 
+    assembly_name: str = Field(..., title="Laser assembly name")
```
Comment (jsiegle) [inline src/aind_data_schema/ephys/ephys_rig.py:127]: Change to `laser_assembly_name` for consistency

**Review by jsiegle — APPROVED** [review]
Just one minor change needed. Otherwise I think this looks good!

**Comment by dyf** [comment]
@jsiegle is a "module" the same thing as an "assembly"? if so, can we just use the term "assembly"?

**Comment by jsiegle** [comment]
We like using the term "module" to refer to a module on the dome insertion system, whereas "assembly" is more general (i.e., a `CameraAssembly` can be part of the behavior setup).

**Pull request merged**